src/node_builtins.cc

```diff
@@ -388,53 +388,57 @@ void BuiltinLoader::SaveCodeCache(const char* id, Local<Function> fun) {
 MaybeLocal<Function> BuiltinLoader::LookupAndCompile(Local<Context> context,
                                                      const char* id,
                                                      Realm* optional_realm) {
-  std::vector<Local<String>> parameters;
   Isolate* isolate = context->GetIsolate();
   // Detects parameters of the scripts based on module ids.
   // internal/bootstrap/realm: process, getLinkedBinding,
   //                           getInternalBinding, primordials
   if (strcmp(id, "internal/bootstrap/realm") == 0) {
-    parameters = {
+    Local<String> parameters[] = {
         FIXED_ONE_BYTE_STRING(isolate, "process"),
         FIXED_ONE_BYTE_STRING(isolate, "getLinkedBinding"),
         FIXED_ONE_BYTE_STRING(isolate, "getInternalBinding"),
         FIXED_ONE_BYTE_STRING(isolate, "primordials"),
     };
-  } else if (strncmp(id,
-                     "internal/per_context/",
-                     strlen("internal/per_context/")) == 0) {
+    std::span<Local<String>> span(&parameters[0], arraysize(parameters));
+    return LookupAndCompileInternal(context, id, span, optional_realm);
+  }
+
+  if (strncmp(id, "internal/per_context/", strlen("internal/per_context/")) ==
+      0) {
     // internal/per_context/*: global, exports, primordials
-    parameters = {
+    Local<String> parameters[] = {
         FIXED_ONE_BYTE_STRING(isolate, "exports"),
         FIXED_ONE_BYTE_STRING(isolate, "primordials"),
     };
-  } else if (strncmp(id, "internal/main/", strlen("internal/main/")) == 0 ||
-             strncmp(id,
-                     "internal/bootstrap/",
-                     strlen("internal/bootstrap/")) == 0) {
+    std::span<Local<String>> span(&parameters[0], arraysize(parameters));
```

Comment:
Shouldn't this be LocalVector<String>?

Reply:
Not here, no. It's just a span over the array.